### PR TITLE
ENT-1682: Update build process to use Fedora Zanata

### PR DESCRIPTION
**Pull request by Januson**
- Switched url to fedora zanata
- Removed deprecated locales block

### CODE REVIEW

File: po/zanata.xml [hunk 1]
```diff
@@ -1,18 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <config xmlns="http://zanata.org/namespace/config/">
-    <url>https://vendors.zanata.redhat.com/</url>
+    <url>https://fedora.zanata.org/</url>
     <project>subscription-manager</project>
+    <project-version>RHEL-7.8</project-version>
```
Comment (cnsnyder) [inline po/zanata.xml]: Let's leave the version out, makes it easier to update the translations via the cli (as we can specify which version at push and pull time as well)

File: syspurpose/po/zanata.xml [hunk 2]
```diff
@@ -1,18 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <config xmlns="http://zanata.org/namespace/config/">
-    <url>https://vendors.zanata.redhat.com/</url>
+    <url>https://fedora.zanata.org/</url>
     <project>syspurpose</project>
+    <project-version>RHEL-7.8</project-version>
```
Comment (cnsnyder) [inline syspurpose/po/zanata.xml]: Let's leave the version out, makes it easier to update the translations via the cli (as we can specify which version at push and pull time as well)

**Comment by cnsnyder** [comment]
@Januson Thank you very much for the PR! I appreciate your help as we transition our process to the new zanata!

In addition to the above could you please rebase your branch on master? That should fix the failing sles tests (which I would really appreciate).

File: po/zanata.xml (hunk 1 above)
Comment (Januson) [reply po/zanata.xml]: Done.

File: syspurpose/po/zanata.xml (hunk 2 above)
Comment (Januson) [reply syspurpose/po/zanata.xml]: Done.

**Comment by wottop** [comment]
buildbot: retest this please

**Pull request merged**